workspaces/optic-engine-wasm/scripts/watch.js

```diff
@@ -0,0 +1,36 @@
+const createWasmPackPlugin = require('../webpack.plugin');
+
+// We provide a mock for the webpack compiler, so we don't have to reimplement all the watching and can
+// leverage the existing plugin.
+
+let beforeCompileHook;
+let thisCompilationHook;
+
+const compiler = {
+  options: {
+    mode: 'development',
+  },
+
+  watchMode: true,
+
+  hooks: {
+    beforeCompile: {
+      tapPromise(pluginName, hook) {
+        beforeCompileHook = hook;
+      },
+    },
+    thisCompilation: {
+      tap(pluginName, hook) {
+        thisCompilationHook = hook;
+      },
+    },
+  },
+};
+
+let wasmPack = createWasmPackPlugin();
+
+wasmPack.apply(compiler);
+
+beforeCompileHook().catch((err) => {
+  throw err;
+});
```

Comment:
How hacky is this? Pretty hacky! But given how incredibly small the integration between the wasm-pack watcher is, there being no alternatives and the complexity of forking / writing it ourselves being more complicated than this, I say it's worth it (for now, at least).